src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
```

Comment:
You have to admit that in most cases it's unnecessary with oop (and even more easily abused)…
But I still think for a case like the one here it's the best way…
As I said: the other possibility would be code doubling and that paves the way to errors…

Not talking about truly switching to oop, which is a lot more work than what I've done here and much more errorprone if you can't test yourself…